base/namedtuple.jl

```diff
@@ -116,6 +119,8 @@ firstindex(t::NamedTuple) = 1
 lastindex(t::NamedTuple) = nfields(t)
 getindex(t::NamedTuple, i::Int) = getfield(t, i)
 getindex(t::NamedTuple, i::Symbol) = getfield(t, i)
+@inline getindex(t::NamedTuple, idxs::Tuple{Vararg{Symbol}}) = NamedTuple{idxs}(t)
```

Comment:
Sorry if this has been discussed elsewhere, but why the `@inline`? Seems to not match the other methods around and should be simple enough inline by itself?

Reply:
That's how Jeff wrote it in the attached issue (#38834) - TBH I still don't totally understand what `@inline` does :shrug: . Happy to leave or remove